test-suites/tss-tests/src/main/java/org/opennms/integration/api/v1/timeseries/InMemoryStorage.java

```diff
@@ -149,7 +154,7 @@ public String toString() {
     }
 
     /** Exposes the internal data, for validation in tests. */
-    public Map<Metric, Collection<Sample>> getData() {
```

Comment:
If this is meant to be internal-only, should we remove `public` from it, and be more explicit in the comment? Otherwise this technically breaks ABI in 1.x.

Reply:
Oh wait, duh, this is only test suite code. Never mind. I should look at paths first before commenting. :D

Reply:
The InMemoryStorage is used in InMemoryStorageTest that tests the functionality of the integration test suit for a plugin.
This method is actually never used. I suggest we just remove it?